agent-control/src/agent_control/defaults.rs

```diff
@@ -1,17 +1,27 @@
 use crate::agent_type::agent_type_id::AgentTypeID;
+use crate::cli::install::agent_control::AGENT_CONTROL_DEPLOYMENT_RELEASE_NAME;
+use crate::cli::install::flux::AGENT_CONTROL_CD_RELEASE_NAME;
 use crate::opamp::remote_config::signature::SIGNATURE_CUSTOM_CAPABILITY;
 use crate::sub_agent::identity::AgentIdentity;
 use opamp_client::capabilities;
 use opamp_client::opamp::proto::{AgentCapabilities, CustomCapabilities};
 use opamp_client::operation::capabilities::Capabilities;
 
 pub const AGENT_CONTROL_ID: &str = "agent-control";
+
+pub const RESERVED_AGENT_IDS: [&str; 3] = [
+    AGENT_CONTROL_ID,
+    AGENT_CONTROL_CD_RELEASE_NAME,
+    AGENT_CONTROL_DEPLOYMENT_RELEASE_NAME,
+];
+
 pub const AGENT_CONTROL_TYPE: &str = "com.newrelic.agent_control";
 pub const AGENT_CONTROL_NAMESPACE: &str = "newrelic";
 pub const AGENT_CONTROL_VERSION: &str = env!("CARGO_PKG_VERSION");
 
 // Keys identifying attributes
 pub const OPAMP_CHART_VERSION_ATTRIBUTE_KEY: &str = "chart.version";
+pub const OPAMP_K8S_CD_VERSION_ATTRIBUTE_KEY: &str = "cd.chart.version";
```

Comment:
should we change as well the one above to `ac.chart.version`?

Reply:
Also in the dynamic config something line "pub ac_chart_version: Option<String>," might be more consistent

Reply:
Hmm did not want to change the existing one to not force FC into a change that might not be trivial for them. Perhaps I can bring it to the ongoing discussion and change in a follow up PR.

Reply:
> already used for sub-agents

🤔 that is true.. but having cd.chart.version than isn't going to be an issue?

should we use subcomponents?